tests/configs/wdio.local.autosave.conf.js

```diff
@@ -0,0 +1,42 @@
+const { join } = require('path');
```

Comment:
Awesome that you created tests, I'm very thankful for this. When I think of this I think it's better to introduce something like a UT framework instead of running this through an instance of WebdriverIO for all methods.

For now I'll keep it like you provided, but for the future I'll change this to UT's and also add proper UT's the `index.js` file which I lacked to do.